cmd/minishift/cmd/start.go

```diff
@@ -347,7 +347,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 
 	cacheMinishiftISO(machineConfig)
 
-	fmt.Print("-- Starting Minishift VM ...")
+	fmt.Printf("-- Starting Minishift VM profile '%s'...", minishiftConfig.AllInstancesConfig.ActiveProfile)
```

Comment:
This is in my opinion too late... several configuration messages have already been shown at this time, such as:

```
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 10 GB
```
And even on Hyper-V.
```
-- Attempting to set network settings ...
   Device:      eth0
   IP Address:  10.0.75.128/24
   Gateway:     10.0.75.1
   Nameservers: 8.8.8.8
```

In my opinion this should be one of the first messages.